### PR TITLE
feat(cmd): Support remove cmd

### LINKED ISSUE #14: [feature] Support remove command to remove the model from the local cache
**Is this a BUG REPORT or FEATURE REQUEST?**:

/kind feature

**What happened**:

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

### PR DESCRIPTION
**Pull request by gaocegege**
Signed-off-by: Ce Gao <gaoce@caicloud.io>

**What this PR does / why we need it**:

Support remove model from the local filesystem cache.

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #14 

Reference to #

**Special notes for your reviewer**:

/cc @ddysher @simon-cj 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/ormb/cmd/remove.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	RunE: func(cmd *cobra.Command, args []string) error {
+		// TODO(gaocegege): Validate.
+		return ormbClient.Remove(args[0])
+	},
+}
+
+func init() {
+	rootCmd.AddCommand(removeCmd)
+
+	removeCmd.Flags().BoolVarP(&plainHTTPOpt, "plain-http", "", false, "use plain http and not https")
```
Comment (simon-cj) [inline cmd/ormb/cmd/remove.go:38]: If it is remove model locally in cache,  plain HTTP option is not MUST, use default `false` is ok.

Comment (gaocegege) [reply cmd/ormb/cmd/remove.go:38]: SGTM, let's remove it.

**Comment by gaocegege** [comment]
@simon-cj Comments are addressed, PTAL

**Pull request merged**